### PR TITLE
(IMAGES-653) Updates to ubuntu-16.04 images

### PR DESCRIPTION
**Pull request by shrug**
Automatic updates are not disabled on our local templates, but the modules here look correct. Rebuild to see if the changes are being applied correctly.

### CODE REVIEW

**Comment by underscorgan** [comment]
@shrug did you test the image build?

**Comment by shrug** [comment]
Locally it looks good. With the vcenter issues I have not been able to deploy a build to the pooler yet.

**Comment by underscorgan** [comment]
fair. I'm gonna approve this, not sure if you want it to be blocked until we can deploy to the pooler.

**Comment by ScottGarman** [comment]
@shrug, we've done a bunch of restructuring of the image templates since you submitted this. Let me know if you'd like me to generate a new PR from this so that follows the new structure.

**Comment by shrug** [comment]
Yeah I was looking at that. Although it looks like the old-style files are still in the repo? Regardless, I was looking at reworking to fit the new format but if you beat me to it... :)

**Comment by ScottGarman** [comment]
Yeah, the manifest files are still in the same locations, but the ubuntu templates themselves have been moved around and split up into an inheritance hierarchy. See the README.md file in the ubuntu directory for instructions on how to build the templates now. I'd be curious to hear feedback on the changes. If you need any help with this don't hesitate to ask and I can work on this next week.

**Comment by shrug** [comment]
This is updated to match the new layout, the jenkins jobs are not updated though and won'yt build this yet

**Comment by ScottGarman** [comment]
Yep, we're about to start making changes to the jenkins imaging pipeline scripts to support the great refactoring. If this builds for you locally using the same build instructions in the top-level debian/README.md, then I'm good to merge this when you're ready (I'm still seeing the don't merge label on this).

:+1:

File: templates/linux/common/vmware.vsphere.nocm.json [hunk 1]
```diff
@@ -25,6 +25,15 @@
     "packer_vcenter_folder"                 : "{{env `PACKER_VCENTER_FOLDER`}}",
     "packer_vcenter_net"                    : "{{env `PACKER_VCENTER_NET`}}",
     "packer_vcenter_insecure"               : "{{env `PACKER_VCENTER_INSECURE`}}",
+    "packer_vcenter2_host"                  : "{{env `PACKER_VCENTER2_HOST`}}",
```
Comment (ekinanp) [inline templates/linux/common/vmware.vsphere.nocm.json]: Unfortunately, Debian platforms have their own vmware.base and vmware.vsphere templates (due to limitations with Packer, not b/c the structure of those templates differ from what's in templates/linux/common). So I think you will also need to include these changes in the corresponding templates/linux/debian/common/vmware.vsphere.nocm.json or whatever. That should be the only one I can think of off the top of my head, as Sol does not yet have a vmware vsphere template and the templates fail to build for OSX 10.12.

**Comment by ScottGarman** [comment]
@ekinanp any last requirements before I merge this? I'm okay with further tweaks as necessary afterward if it's not 100% perfect, as I'd prefer not see this PR hang around much longer.

**Comment by ekinanp** [comment]
@ScottGarman Nope, feel free to merge it.

**Pull request merged**